sdk/resourcemanager/azure-resourcemanager-network/src/main/java/com/azure/resourcemanager/network/models/LoadBalancerOutboundRule.java

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.network.models;
+
+import com.azure.resourcemanager.network.fluent.models.OutboundRuleInner;
+import com.azure.resourcemanager.resources.fluentcore.arm.models.ChildResource;
+import com.azure.resourcemanager.resources.fluentcore.model.Attachable;
+import com.azure.resourcemanager.resources.fluentcore.model.HasInnerModel;
+import com.azure.resourcemanager.resources.fluentcore.model.Settable;
+
+import java.util.List;
+import java.util.Map;
+
+/** An immutable client-side representation of an outbound rule. */
+public interface LoadBalancerOutboundRule
+    extends HasInnerModel<OutboundRuleInner>,
+    HasProtocol<LoadBalancerOutboundRuleProtocol>,
+    ChildResource<LoadBalancer> {
+
+
+    /** @return the associated frontend IP configuration ids */
+    List<String> frontendIpConfigurationIds();
+
+    /** @return the associated frontends */
+    Map<String, LoadBalancerFrontend> frontends();
+
+    /** @return the associated backend address pool id */
+    String backendAddressPoolId();
+
+    /** @return the associated backend */
+    LoadBalancerBackend backend();
```

Comment:
We might not need `frontendIpConfigurationIds` and `backendAddressPoolId` getter, if there is no extra HTTP request required in `frontends` and `backend`.